kodilla-collections/src/main/java/com/kodilla/collections/interfaces/homework/Ford.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.collections.interfaces.homework;
+
+public class Ford implements Car {
+
+    private double speed = 150;
+    private double acceleration = 15;
+    private double deceleration = 20;
+
+    public double getSpeed() {
+        return speed;
+    }
+
+    public void increaseSpeed() {
+        speed += acceleration;
+    }
+
+    public void decreaseSpeed() {
+        speed -= deceleration;
```

Comment:
Czy predkosc moze byc ujemna? :) Jak naciskam pedal hamulca z calej sily, to raczej do tylu nie pojade